lib/java/test/org/apache/thrift/test/TestClient.java

```diff
@@ -576,7 +581,7 @@ public static void main(String [] args) {
           goodbye.string_thing = "Goodbye4";
           goodbye.byte_thing = (byte)4;
           goodbye.i32_thing = 4;
-          goodbye.i64_thing = (long)4;
+          goodbye.i64_thing = 4;
 
```

Comment:
Yes that is unrelated... I think my IDE settings are such that it was removed automatically because it was an unncessary... no problem for me to restore it if you prefer.